pkg/host/internal/sriov/sriov_test.go

```diff
@@ -122,6 +121,7 @@ var _ = Describe("SRIOV", func() {
 			netlinkLibMock.EXPECT().LinkByName("enp216s0f0np0").Return(pfLinkMock, nil).Times(3)
 			pfLinkMock.EXPECT().Attrs().Return(&netlink.LinkAttrs{OperState: netlink.OperDown, EncapType: "ether"}).Times(2)
 			netlinkLibMock.EXPECT().LinkSetUp(pfLinkMock).Return(nil)
+			netlinkLibMock.EXPECT().LinkList().Return(nil, nil).AnyTimes()
```

Comment:
Can you remove usage of `AnyTimes` in the tests and use expected call count explicitly? I think this is important to validate that function flow works as expected.

Reply:
Done